.buildkite/pipeline.yml

```
@@ -31,6 +31,20 @@ steps:
       - github_commit_status:
           context: Prototype Build
 
+  - group: ":rocket: Prototype Build"
+    if: build.branch == "trunk"
+    steps:
+      - input: Deploy Prototype Build
```

💄 Nitpick: I feel like adding a question mark to the name of the step helps making it more obvious in the Buildkite Canvas UI that this is a step requiring human approval (especially to devs less familiar with that feature existing in Buildkite, who might not realize that the purple arrow in that UI means "awaiting for human to answer").


```suggestion
      - input: Deploy Prototype Build?
```